code/game/objects/items/circuitboards/computer_circuitboards.dm

```diff
@@ -47,6 +47,19 @@
 	build_path = /obj/machinery/computer/communications
 	var/lastTimeUsed = 0
 
+/obj/item/circuitboard/computer/communications/Initialize(mapload)
+	. = ..()
+	GLOB.shuttle_caller_list += src
+
+/obj/item/circuitboard/computer/communications/obj_break()
+	SSshuttle.autoEvac()
+	. = ..()
+
+/obj/item/circuitboard/computer/communications/Destroy(mapload)
+	GLOB.shuttle_caller_list -= src
+	SSshuttle.autoEvac()
+	..()
```

Comment:
This is a bad idea in general, since there are many places where the boards can be that are not destroyed but still inaccessible to crew, not to mention that there may very well be no power to power such a console even if they have the board.

Leave it at the roundstart consoles + AI, this is already rarely invoked.

Reply:
Similarly, assuming I read the code correctly, the R&D protolathe shouldn't be counting as a shuttle caller, since a) it doesn't even print the circuit boards (you want the circuit printer) and b) the above